### ISSUE TITLE
Table cell width / alignment issues in some scenarios.

### ISSUE THREAD

**Issue by rwjblue** [kategengler/ember-try#46]
Example from a `ember try:testall` in `ember-test-helpers`:

```
Scenario ember-release: SUCCESS
┌────────────────────┬────────────────────┬──────────────────────────────┬──────────┐
│ Dependency         │ Expected           │ Used                         │ Type     │
├────────────────────┼────────────────────┼──────────────────────────────┼──────────┤
│ ember              │ components/e… │ 2.3.0+941794ac               │ bower    │
├────────────────────┼────────────────────┼──────────────────────────────┼──────────┤
│ ember-data         │ ~2.2.0             │ 2.2.1                        │ bower    │
└────────────────────┴────────────────────┴──────────────────────────────┴──────────┘

Scenario ember-beta: SUCCESS
┌────────────────────┬────────────────────┬──────────────────────────────┬──────────┐
│ Dependency         │ Expected           │ Used                         │ Type     │
├────────────────────┼────────────────────┼──────────────────────────────┼──────────┤
│ ember              │ components/e… │ 2.4.0-beta.1+7fbca9c0        │ bower    │
├────────────────────┼────────────────────┼──────────────────────────────┼──────────┤
│ ember-data         │ ~2.2.0             │ 2.2.1                        │ bower    │
└────────────────────┴────────────────────┴──────────────────────────────┴──────────┘
```

**Comment by rwjblue** [comment]
(BTW, in general this table formatting is really awesome!)

**Comment by rwjblue** [comment]
Seems like the ones that are misaligned as the ones that have been truncated, might be a good place to dig...

**Comment by kategengler** [comment]
The npm package `cli-table` is making the table. While looking for it again, I came across https://www.npmjs.com/package/cli-table2, which has as one of its features "More robust truncation of cell text that contains ansi color characters.". Since ember-try is coloring the output, I feel this strongly implies just changing to cli-table2 might fix it. Will check later tonight, or tomorrow.

**Comment by kategengler** [comment]
`cli-table2` does not seem to fix it.

**Comment by rwjblue** [comment]
Hmm, I was just trying also.  I pushed my changes to https://github.com/kategengler/ember-try/compare/master...rwjblue:cli-table2.

The output from `ember try ember-release` with these changes linked, does look correct:

```
Scenario ember-release: SUCCESS
┌────────────────────┬────────────────────┬──────────────────────────────┬──────────┐
│ Dependency         │ Expected           │ Used                         │ Type     │
├────────────────────┼────────────────────┼──────────────────────────────┼──────────┤
│ ember              │ components/ember#… │ 2.3.0+667a2a6b               │ bower    │
├────────────────────┼────────────────────┼──────────────────────────────┼──────────┤
│ ember-data         │ ~2.2.0             │ 2.2.1                        │ bower    │
└────────────────────┴────────────────────┴──────────────────────────────┴──────────┘
```

**Comment by kategengler** [comment]
Ah, I forgot to change the require. Ah well, next release!

**Comment by kategengler** [comment]
Fixed by #51
